kaldo/observables/thirdorder.py

```diff
@@ -168,10 +168,11 @@ def load(cls, folder, supercell=(1, 1, 1), format='sparse', third_energy_thresho
             replicated_atom_prime_file = str(folder) + '/' + replicated_filename
             atoms = ase.io.read(atom_prime_file)
             replicated_atoms = ase.io.read(replicated_atom_prime_file)
-            with open('FC3', 'r') as f:
+            file_name = str(folder) + '/' + 'FC3'
+            with open(file_name, 'r') as f:
                 lines = f.readlines()
                 f.close()
-            c = 13.605693012183622 * (1.889725989 ** (3))
+            c = units.Rydberg * ((1/units.Bohr) ** (3)) # conversion factor from Ry/bohr^3 to eV/A^3
```

Comment:
Minor, no need for `(value)`. It should be `value`, otherwise Python creates a tuple.